id-repository/credential-service/src/main/java/io/mosip/credentialstore/util/EncryptionUtil.java

```diff
@@ -213,7 +217,8 @@ public String encryptData(String dataToBeEncrypted, String partnerId, String req
 				ServiceError error = responseObject.getErrors().get(0);
 				throw new DataEncryptionFailureException(error.getMessage());
 			}
-			encryptedPacket = responseObject.getResponse().getData();
+			encryptedPacket = responseObject!=null? responseObject.getResponse().getData() : null;

```

Comment:
@vipuldhurve use Objects.nonNull instead of != and also add null check for responseObject.getResponse() also